qiskit/circuit/add_control.py

```diff
@@ -14,22 +14,27 @@
 """
 Add control to operation if supported.
 """
+
+from typing import List, Optional, Union, TYPE_CHECKING
```

Comment:
No need to import List, it is not used. (Just dropped by at your file to see an example of using `typing`, it is very helpful, thanks :smile: ).

Reply:
thanks!